elastic-container.sh

```diff
@@ -15,92 +15,185 @@
 # More information at https://github.com/peasead/elastic-container"
 
 # Define variables
+ELASTIC_USERNAME="elastic"
```

Comment:
Why are we defining a username?

Reply:
It's required for basic auth with curl so we can enable the detection engine and prebuilt rules

Reply:
Why don't we just use `elastic` instead of setting another variable?

Reply:
Because it's cleaner and more efficient this way. If we ever changed the username or a user wanted to they would only have to do so in one place instead of within every curl command.